docs/dev_guide.rst

```diff
@@ -0,0 +1,110 @@
+.. _dev_guide:
+
+*****************
+Developer's Guide
+*****************
+
+People who wish to contribute to this package should fork the `GitHub repository <https://github.com/Helioviewer-Project/python-api>`__.
+
+You need to install the development version of the package in order to make and test changes.
+See the :ref:`obtaining_the_source` section for details.
+
+Building Documentation
+----------------------
+
+.. note::
+
+    Building the documentation is not necessary unless you are writing new documentation or do not have internet access, because the latest versions of ``hvpy``'s should be available at `hvpy.readthedocs.io <https://hvpy.readthedocs.io/>`__.
+
+Dependencies
+^^^^^^^^^^^^
+
+Building the documentation requires the ``hvpy`` source code and some additional packages.
+The easiest way to build the documentation is to use `tox <https://tox.readthedocs.io/en/latest/>`__ as detailed in :ref:`hvpy-doc-building`.
+
+.. note::
+
+    This does not include `Graphviz <http://www.graphviz.org>`__.
+    If you do not install this package separately then the documentation build process will produce a very large number of lengthy warnings (which can obscure bona fide warnings) and also not generate inheritance graphs.
+
+.. _hvpy-doc-building:
```

Comment:
This should be moved to above the `Building Documentation` title and not this one.